src/components/NavBar/JumpToPursuance/JumpToPursuance.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { DropdownButton, MenuItem } from 'react-bootstrap';
+import { setCurrentPursuance } from '../../../actions';
+
+const getCurrentPursuanceName = (pursuances, currentPursuanceId) => {
+  const rawPursuance = pursuances[currentPursuanceId];
+  if (rawPursuance !== undefined) {
+    return rawPursuance.name;
+  } else {
+    return "Jump to a pursuance";
```

Comment:
`return "Jump to a " + PROJECT;`, please

Reply:
See https://github.com/EffectiveAF/effective/commit/136c5710c8de5b91f183a16e51f20f6ee9a98ea9